code/modules/reagents/reagent_containers/food/drinks/bottle.dm

```diff
@@ -267,10 +281,18 @@
 	//The reagents in the bottle splash all over the target, thanks for the idea Nodrak
 	var/alcohol_power = calculate_alcohol_power()
 
-	if (reagents)
+
+// Checks if there are reagents in the bottle to output the message of us splashing alcohol over the target.
+	if (reagents && reagents.total_volume > 0)
+		reagents.splash(target, reagents.total_volume)
 		spawn (1) // wait until after our explosion, if we have one
 			user.visible_message("<span class='notice'>The contents of \the [src] splash all over [target]!</span>")
-			if (reagents) reagents.splash(target, reagents.total_volume)
+			shatter(loc, target, alcohol_power)
```

Comment:
Same problem as in previous comments. Might even want to do the same for the spawn(1).